### PR TITLE
cp: add Nepali translation

### PR DESCRIPTION
**Pull request by psycho-arna**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — CHANGES_REQUESTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Translate paths inside `{{}}`**, you can refer to [common arguments](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md) if necessary.

File: pages.ne/common/cp.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# cp
+
+> फाइलहरू र डिरेक्टोरीहरू सार्नुहोस।
+> थप जानकारी: <https://www.gnu.org/software/coreutils/cp>
+
+- एउटा स्थान बाट अर्को स्थानमा फाइलहरु सार्नुहोस :
+
+`cp {{स्रोत/फाइलको/बाटो}} {{लक्ष्य/फाइलको/बाटो}}`
```
Comment (kbdharun) [inline pages.ne/common/cp.md:8]:
```suggestion
`cp {{स्रोत_फाइल.ext/को/पथ}} {{लक्ष्य_फाइल.ext/को/पथ}}`
```

File: pages.ne/common/cp.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- फाइलको नाम उही राखेर अर्को डिरेक्टोरीमा फाइलहरु सार्नुहोस :
+
+`cp {{स्रोत/फाइलको/बाटो}} {{लक्ष्य/डिरेक्टोरीको/बाटो}}`
```
Comment (kbdharun) [inline pages.ne/common/cp.md:12]:
```suggestion
`cp {{स्रोत_फाइल.ext/को/पथ}} {{लक्षित_अभिभावक_निर्देशिका/को/पथ}}`
```

File: pages.ne/common/cp.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- पुनरावर्ती रूपमा डिरेक्टोरीमा भएका सबै चीजहरुलाइ अर्को स्थानमा सार्नुहोस(यदि स्थान पहिले देखि नै छ भने,डिरेक्टोरी भित्र सार्नुहोस ):
+
+`cp -R {{स्रोत/डिरेक्टोरीको/बाटो}} {{लक्ष्य/डिरेक्टोरीको/बाटो}}`
```
Comment (kbdharun) [inline pages.ne/common/cp.md:16]:
```suggestion
`cp -R {{स्रोत_निर्देशिका/को/पथ}} {{लक्ष्य_निर्देशिका/को/पथ}}`
```

File: pages.ne/common/cp.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`cp -vR {{स्रोत/डिरेक्टोरीको/बाटो}} {{लक्ष्य/डिरेक्टोरीको/बाटो}}`
+
+- अन्तरक्रियात्मक रुपमा अर्को स्थानमा पाठ्य फाइलहरू सार्नुहोस (अधिलेखन गर्नु अघि प्रयोगकर्तालाई सोध्छ ):
+
+`cp -i {{*.txt}} {{लक्ष्य/डिरेक्टोरीको/बाटो}}`
+
+- सार्नु अघि प्रतीकात्मक लिङ्कहरू पछ्याउनुहोस्:
+
+`cp -L {{लिङ्क}} {{लक्ष्य/डिरेक्टोरीको/बाटो}}`
```
Comment (kbdharun) [inline pages.ne/common/cp.md:28]:
```suggestion
`cp -L {{लिङ्क}} {{लक्ष्य_निर्देशिका/को/पथ}}`
```

File: pages.ne/common/cp.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+`cp -R {{स्रोत/डिरेक्टोरीको/बाटो}} {{लक्ष्य/डिरेक्टोरीको/बाटो}}`
+
+- पुनरावर्ती रूपमा एउटा डाइरेक्टरी सार्नुहोस, शब्दमय रुपमा (फाइलहरु सार्दा सार्दै सरिरहेको पनि देखिन्छ ):
+
+`cp -vR {{स्रोत/डिरेक्टोरीको/बाटो}} {{लक्ष्य/डिरेक्टोरीको/बाटो}}`
+
+- अन्तरक्रियात्मक रुपमा अर्को स्थानमा पाठ्य फाइलहरू सार्नुहोस (अधिलेखन गर्नु अघि प्रयोगकर्तालाई सोध्छ ):
+
+`cp -i {{*.txt}} {{लक्ष्य/डिरेक्टोरीको/बाटो}}`
```
Comment (kbdharun) [inline pages.ne/common/cp.md:24]:
```suggestion
`cp -i {{*.txt}} {{लक्ष्य_निर्देशिका/को/पथ}}`
```

File: pages.ne/common/cp.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+`cp {{स्रोत/फाइलको/बाटो}} {{लक्ष्य/डिरेक्टोरीको/बाटो}}`
+
+- पुनरावर्ती रूपमा डिरेक्टोरीमा भएका सबै चीजहरुलाइ अर्को स्थानमा सार्नुहोस(यदि स्थान पहिले देखि नै छ भने,डिरेक्टोरी भित्र सार्नुहोस ):
+
+`cp -R {{स्रोत/डिरेक्टोरीको/बाटो}} {{लक्ष्य/डिरेक्टोरीको/बाटो}}`
+
+- पुनरावर्ती रूपमा एउटा डाइरेक्टरी सार्नुहोस, शब्दमय रुपमा (फाइलहरु सार्दा सार्दै सरिरहेको पनि देखिन्छ ):
+
+`cp -vR {{स्रोत/डिरेक्टोरीको/बाटो}} {{लक्ष्य/डिरेक्टोरीको/बाटो}}`
```
Comment (kbdharun) [inline pages.ne/common/cp.md:20]:
```suggestion
`cp -vR {{स्रोत_निर्देशिका/को/पथ}} {{लक्ष्य_निर्देशिका/को/पथ}}`
```

File: pages.ne/common/cp.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
+# cp
+
+> फाइलहरू र डिरेक्टोरीहरू सार्नुहोस।
+> थप जानकारी: <https://www.gnu.org/software/coreutils/cp>
```
Comment (kbdharun) [inline pages.ne/common/cp.md:4]:
```suggestion
> थप जानकारी: <https://www.gnu.org/software/coreutils/cp>.
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. The translation LGTM. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

File: pages.ne/common/cp.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
+# cp
+
+> फाइलहरू र डिरेक्टोरीहरू सार्नुहोस।
+> थप जानकारी: <https://www.gnu.org/software/coreutils/cp>.
+
+- एउटा स्थान बाट अर्को स्थानमा फाइलहरु सार्नुहोस :
```
Comment (navarroaxel) [inline pages.ne/common/cp.md:6]: Is this space required in Nepali?
```suggestion
- एउटा स्थान बाट अर्को स्थानमा फाइलहरु सार्नुहोस:
```

File: pages.ne/common/cp.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+`cp {{स्रोत_फाइल.ext/को/पथ}} {{लक्ष्य_फाइल.ext/को/पथ}}`
+
+- फाइलको नाम उही राखेर अर्को डिरेक्टोरीमा फाइलहरु सार्नुहोस :
```
Comment (navarroaxel) [inline pages.ne/common/cp.md:10]:
```suggestion
- फाइलको नाम उही राखेर अर्को डिरेक्टोरीमा फाइलहरु सार्नुहोस:
```

File: pages.ne/common/cp.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+
+`cp {{स्रोत_फाइल.ext/को/पथ}} {{लक्षित_अभिभावक_निर्देशिका/को/पथ}}`
+
+- पुनरावर्ती रूपमा डिरेक्टोरीमा भएका सबै चीजहरुलाइ अर्को स्थानमा सार्नुहोस(यदि स्थान पहिले देखि नै छ भने,डिरेक्टोरी भित्र सार्नुहोस ):
```
Comment (navarroaxel) [inline pages.ne/common/cp.md:14]:
```suggestion
- पुनरावर्ती रूपमा डिरेक्टोरीमा भएका सबै चीजहरुलाइ अर्को स्थानमा सार्नुहोस(यदि स्थान पहिले देखि नै छ भने,डिरेक्टोरी भित्र सार्नुहोस):
```

File: pages.ne/common/cp.md [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+
+`cp -R {{स्रोत_निर्देशिका/को/पथ}} {{लक्ष्य_निर्देशिका/को/पथ}}`
+
+- पुनरावर्ती रूपमा एउटा डाइरेक्टरी सार्नुहोस, शब्दमय रुपमा (फाइलहरु सार्दा सार्दै सरिरहेको पनि देखिन्छ ):
```
Comment (navarroaxel) [inline pages.ne/common/cp.md:18]:
```suggestion
- पुनरावर्ती रूपमा एउटा डाइरेक्टरी सार्नुहोस, शब्दमय रुपमा (फाइलहरु सार्दा सार्दै सरिरहेको पनि देखिन्छ):
```

File: pages.ne/common/cp.md [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+
+`cp -vR {{स्रोत_निर्देशिका/को/पथ}} {{लक्ष्य_निर्देशिका/को/पथ}}`
+
+- अन्तरक्रियात्मक रुपमा अर्को स्थानमा पाठ्य फाइलहरू सार्नुहोस (अधिलेखन गर्नु अघि प्रयोगकर्तालाई सोध्छ ):
```
Comment (navarroaxel) [inline pages.ne/common/cp.md:22]:
```suggestion
- अन्तरक्रियात्मक रुपमा अर्को स्थानमा पाठ्य फाइलहरू सार्नुहोस (अधिलेखन गर्नु अघि प्रयोगकर्तालाई सोध्छ):
```

File: pages.ne/common/cp.md (hunk 8 above)
Comment (kbdharun) [reply pages.ne/common/cp.md:6]: I don't think it is required. We generally don't include space in line endings in Indic languages.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**